codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java

```diff
@@ -91,10 +90,10 @@
  *
  * <pre>{@code
  *     final Http2Stream2 stream = handler.newStream();
- *     ctx.write(headersFrame.stream(stream)).addListener(new ChannelFutureListener() {
+ *     ctx.write(headersFrame.stream(stream)).addListener(new FutureListener() {
  *
  *         @Override
- *         public void operationComplete(ChannelFuture f) {
+ *         public void operationComplete(Future f) {
```

Comment:
I think the `<Void>`is missing here. Also the same is true for `FutureListener` above.